src/text.c

```diff
@@ -163,7 +163,7 @@ static void write_value(int fd, const char *v) {
 }
 
 static bool text_save(Sdb *s, int fd, bool sort, SdbList *path) {
-	//path
+	// path
```

Comment:
this comment can be removed

Reply:
Every comment can be removed, but I would like to keep this one since it splits the function in three distinct sections.